docsite/docsite.css

```diff
@@ -72,7 +74,13 @@ background-color: var(--gray-500)
     border-radius: 3px;
     box-shadow: rgba(0, 0, 0, 0.05) 0px 6px 24px 0px, rgba(0, 0, 0, 0.08) 0px 0px 0px 1px;
 }
-
+.text-code{
+    background-color: var(--gray-200);
+    border-radius: 6px;
+    padding: 3px;
+    font-size: 13px;
```

Comment:
if px is less than 10px you can use it use rem more than 10px.

Reply:
will change that